src/lib/plugins/entities.js

```diff
@@ -1,5 +1,6 @@
 var Entity=require("../entity");
 var Vec3 = require("vec3").Vec3;
+var ItemStack = require("prismarine-item")("1.8")
```

Comment:
use require("../version") instead of "1.8" (so we don't have to change the version number everywhere when we move to 1.9)